packages/amplify-e2e-tests/src/query-utils/gql-helper.ts

```diff
@@ -96,8 +98,24 @@ export class GQLQueryHelper {
     return deleteResult;
   };
 
-  get = async (input: any, selectionSet?: string): Promise<any> => {
-    const finalSelectionSet = selectionSet ?? this.selectionSet.query.get;
+  get = async (
+    input: any,
+    selectionSet?: string,
+    isCompleteQuery = true,
+    errorPolicy?: ErrorPolicy,
```

Comment:
nit: why not assign a default here like you do for the preceding and following args?